orthoinference/src/main/java/org/reactome/orthoinference/EventsInferrer.java

```diff
@@ -113,6 +115,7 @@ public static void inferEvents(Properties props, String species) throws Exceptio
 			Map<String,String[]> homologueMappings = readHomologueMappingFile(species, "hsap", pathToOrthopairs);
 			ProteinCountUtility.setHomologueMappingFile(homologueMappings);
 			EWASInferrer.setHomologueMappingFile(homologueMappings);
+			SkipInstanceChecker.setHomologueMappingFile(homologueMappings);
```

Comment:
The name `setHomologueMappingFile` is a little misleading since it is taking the homologue map data structure itself rather than the file name.  Could it be `setHomologueMap` instead?